apps/api/tests/test_guest.py

```diff
@@ -85,7 +85,9 @@ def test_requesting_login_when_previous_key_exists_causes_429(
 
 
 @patch("auth.guest_auth._get_existing_key", autospec=True)
+@patch("auth.guest_auth._remove_guest_key", autospec=True)
 def test_successful_guest_verification_provides_identity(
+    mock_remove_guest_key: AsyncMock,
     mock_get_existing_key: AsyncMock,
```

Comment:
Suggestion [repeating]: patch order should usually reflect order of usage/calls during test.